controller/space_blackbox_test.go

```diff
@@ -89,6 +89,15 @@ func (s *SpaceControllerTestSuite) SecuredSpaceIterationController(identity acco
 
 func (s *SpaceControllerTestSuite) TestValidateSpaceName() {
 
+	s.T().Run("ok", func(t *testing.T) {
```

Comment:
New test added for Space name validation which will validate space name with exact Max Length (i.e. 63 chars).